### PR TITLE
Support `graphql.experimental` tag

### PR DESCRIPTION
**Pull request by asiandrummer**

### CODE REVIEW

**Comment by wincent** [comment]
For context, this is [used](https://github.com/facebook/relay/blob/15a7ebb5d87f8aae3d1a23ed0cac15f1eff5714e/packages/react-relay/classic/query/RelayGraphQLTag.js#L38-L48) in [Relay](https://github.com/facebook/relay/).

File: grammars/Babel Language.json [hunk 1]
```diff
@@ -935,7 +935,7 @@
         },
         {
           "comment": "GraphQL ( Relay.QL ) supprt. Use two forms of scopes! fixes some themes",
-          "begin": "\\s*+(?:((Relay)(.)(QL))|(gql|graphql)|(/\\* GraphQL \\*/))\\s*((`))",
+          "begin": "\\s*+(?:((Relay)(.)(QL))|(gql|graphql|graphql.experimental)|(/\\* GraphQL \\*/))\\s*((`))",
```
Comment (wincent) [inline grammars/Babel Language.json]: Technically, probably want to backslash-escape the period, right @asiandrummer?

Comment (gandm) [inline grammars/Babel Language.json]: If you add a back slash can you also add one to the Relay.QL regex as that's an error on my part.

**Comment by asiandrummer** [comment]
@gandm how about now?

**Comment by gandm** [comment]
@asiandrummer The '\' itself should be escaped as it is going inside a JSON string. so `\\.` will do it.

**Comment by asiandrummer** [comment]
@gandm I'd also appreciate a lot if you can cut another release with this PR soon ;)

**Comment by gandm** [comment]
@asiandrummer Just published it in 2.58.0

**Pull request merged**